kimchi/cairo/src/runner/definitions.rs

```diff
@@ -0,0 +1,72 @@
+/// Defining some constants for easier readability of the steps
+/// When they refer to single bit flagsets, only one constant is needed
+
+/*pub enum FlagBit {
+    DSTAP = 0,
+    OP0AP = 0,
+    OP1DBL = 0,
+    OP1VAL = 1,
+    OP1FP = 2,
+    OP1AP = 4,
+    RESONE = 0,
+    RESADD = 1,
+    RESMUL = 2,
+    PCSIZ = 0,
+    PCABS = 1,
+    PCREL = 2,
+    PCJNZ = 4,
+    APZ2 = 0,
+    APADD = 1,
+    APONE = 2,
+    OPCJMPINC = 0,
+    OPCCALL = 1,
+    OPCRET = 2,
+    OPCAEQ = 4,
+}*/
```

Comment:
delete?

Reply:
Oh yes! This was an attempt to represent those constants in a more elegant way using enums as Joseph suggested me the other day. But I probably didn't imagine the same type of solution as he was, and indeed the commented lines do not work (as rust does not allow repeated values in an enum). So by now, I will keep those as constants, as their only goal is to increase readability when defining the step behaviour, as opposed to just writing the integers (which you may not remember what they correspond to).